basalt/objects/dataset.py

```
@@ -0,0 +1,92 @@
+    """
+    values: Dict[str, str] 
+    name: Optional[str] = None
+    ideal_output: Optional[str] = None
+    metadata: Dict[str, Any] = field(default_factory=dict)
+
+    def to_dict(self) -> Dict[str, Any]:
+        """Convert the DatasetRow to a dictionary for API requests"""
+        result = {
+            "values": self.values,
+            "metadata": self.metadata,
+						"name": self.name,
```

Flake8 (7.2.0)

[error] indentation contains mixed spaces and tabs

(E101)